### ISSUE TITLE
Using Paraphrase Mining for different Lists

### ISSUE THREAD

**Issue by e412** [UKPLab/sentence-transformers#1500]
Hi to the Sentence-Transformers Team :)

is there a way to implement the Paraphrase Minig Model for 2 different lists so each sentence in one List is not compared to each other sentence but only to all the sentences from the second list?

Thanks in Advance 

Cheers, Eva

**Comment by nreimers** [comment]
Yes, a simple modification of paraphrase_mining function would be sufficient.

You can also check:
https://www.sbert.net/examples/applications/parallel-sentence-mining/README.html

Which also works when your two sentence collections are in the same language

**Comment by e412** [comment]
Thanks. I modified the function. Is it reasonable that my top10 Lists and also scores differ from the matches I found using the Semantic Textual Similarity for the same Dataset? I am wondering because they should be both using the Cosin Similarity...

**Comment by robinsonkwame** [comment]
What is the modification to `paraphrase_mining`?
